operator/k8s_cep_gc.go

```diff
@@ -90,17 +91,41 @@ func enableCiliumEndpointSyncGC(once bool) {
 					// will delete all CEPs in the cluster regardless of the pod
 					// state therefore we won't even watch for the pod store.
 					if !once {
-						podObj, exists, err := watchers.PodStore.GetByKey(cepFullName)
-						if err != nil {
-							scopedLog.WithError(err).Warn("Unable to get pod from store")
-							continue
+						var podObj interface{}
```

Comment:
Cannot comment on the entire `DoFunc`, so adding this here:

Maybe move the entire logic to a named `func` in a follow-up commit/PR? The length and the indentation level of this is becoming quite large, so factoring it into a named `func` would improve readability IMO.

Reply:
Done.